metadata-ingestion/tests/unit/test_bigqueryv2_usage_source.py

```diff
@@ -147,11 +94,8 @@ def test_bigqueryv2_filters_with_deny_filter():
         AND protoPayload.metadata.jobChange.job.jobStats.queryStats.referencedTables:*
         AND NOT protoPayload.metadata.jobChange.job.jobStats.queryStats.referencedTables =~ "projects/.*/datasets/.*/tables/__TABLES__|__TABLES_SUMMARY__|INFORMATION_SCHEMA.*"
          AND (
-            protoPayload.metadata.jobChange.job.jobStats.queryStats.referencedTables =~ \"projects/.*/datasets/.*/tables/test-regex|test-regex-1\"
-            AND
-            NOT (
-                protoPayload.metadata.jobChange.job.jobStats.queryStats.referencedTables =~ \"projects/.*/datasets/.*/tables/excluded_table_regex|excluded-regex-2\"
-            )
+            
+            FALSE
```

Comment:
So if you look at https://cloud.google.com/logging/docs/view/logging-query-language#syntax_summary, I believe that `FALSE` here ends up being a "single value" and not a boolean expression. Because there's no operator (e.g. `=`) in the expression `FALSE`, it gets interpreted as a single value. This effectively invalidates this boolean expression because I guess all log entries contain a FALSE somewhere. Maybe FALSE gets casted into empty string and then every log entry has empty string? Idk. But in any case, if we do want the filter on job reason, we should get rid of the FALSE OR.

Also, looking at the query spec, I'm very skeptical of the AND NOT regex above this line... I think we need parentheses around `(__TABLES__|__TABLES_SUMMARY__|INFORMATION_SCHEMA)`. It still technically works because `INFORMATION_SCHEMA.*` ends up being a valid sql, as does `projects/.*/datasets/.*/tables/__TABLES__` I guess.

Lastly, pretty sure all these escaped quotes do nothing -- in python, `'\"' == '"'`. If we wanted to escape a quote, we'd have to do `\\"`, but I don't think we ever need to.

Overall this is just convincing me we need to take a serious look at these filters (and ideally test them)